packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -202,13 +221,36 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
                     color={color}
                   />
                 )}
-                <FocusZone>
-                  <div className={classNames.entityHost} data-is-focusable={true} onFocus={this._onFocusHandler}>
-                    <div className={classNames.entityInnerHost} ref={this._entityInnerHostElement}>
+                <FocusTrapZone isClickableOutsideFocusTrap={true} forceFocusInsideTrap={false}>
+                  <div
+                    className={classNames.entityHost}
+                    tabIndex={-1}
+                    data-is-focusable={true}
+                    role="dialog"
+                    aria-labelledby={ariaLabelledBy}
+                    aria-describedby={ariaDescribedBy}
+                  >
+                    {isCollapsed &&
+                      ariaLabelledBy && (
+                        <p id={ariaLabelledBy} className={classNames.ariaContainer}>
+                          {ariaLabelledByText}
+                        </p>
+                      )}
+                    {isCollapsed &&
+                      ariaDescribedBy && (
+                        <p id={ariaDescribedBy} className={classNames.ariaContainer}>
```

Comment:
You can move this up to the other isCollapsed section so you don't need to evaluate twice. You might need to make it an array if it complains about needing a single parent but I don't believe you'll need to do that.

Reply:
Fixed in latest commit